sync/bug.py

```diff
@@ -93,6 +94,10 @@ def _get_bug(self, bug_id):
             except bugsy.BugsyException:
                 logger.error("Failed to retrieve bug with id %s" % bug_id)
                 return
+            except Exception as e:
```

Comment:
I'm not sure what to do in this case; I see we already return None in some cases, but I kind of doubt that all the callers can really handle it. But maybe we should do that for now, plus log to NR, and fix this better when we are doing all the bug updates in a single place.